legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/dbExtension.pure

```diff
@@ -715,3 +715,16 @@ Class meta::relational::functions::sqlQueryToString::ToSql
    transform:meta::pure::metamodel::function::Function<{String[*]->String[*]}>[0..1];
    parametersWithinWhenClause : Boolean[*];
 }
+
+
+function meta::relational::functions::sqlQueryToString::processSplitPart():LambdaFunction<{String[3]->String[1]}>[1]
```

Comment:
move this fn to defaultExtension.pure . dbExtension.pure is only for api, for shared fns across <dbType>Extension.pure, use defaultExtension.pure